docs/experience/user-satisfaction.md

```diff
@@ -0,0 +1,67 @@
+title: User Satisfaction
+description: How user satisfaction is measured in Sematext Experience
+
+On the Overview page of your Experience app you will notice several scores that show the overall user satisfaction for the related performance section of your website.
+
+<img
+  class="content-modal-image"
+  alt="Sematext Experience Satisfaction Scores"
+  src="/images/experience/satisfaction.png"
```

Comment:
Add a relative path here, I'm 99% sure the image will break like this. 
`src="../images/experience/satisfaction.png"`

Reply:
I'll try that again, for some reason relative path didn't work.

Reply:
Seems `../../images` is required because it's one more directory level...